trams_api_spec/trams_api_spec.yaml

```diff
@@ -834,6 +854,36 @@ definitions:
         type: "string"
       status:
         type: "string"
+  AcademyTransferProjectSearch:
+    description: "Listing academy transfer projects"
+    type: "object"
+    properties: 
+      project_urn:
+        type: "string"
+        description: "Unique reference for a project"
+      outgoing_trust_urn:
```

Comment:
URN, UKPRN, both?

Reply:
I changed this to UKPRN - i was keeping it consistent with the other references to URN but I _think_ we should be using UKPRN? Happy to defer to you here though

Reply:
trusts should only have UKPRN (most of the time) or GroupID in the case of no UKPRN
 -  are you masking that here and having `unique reference number` for the trust be whichever one you have? feels dangerous. I'd put both UKPRN and GroupID in the trust bit
academies should have a UKPRN and a URN as a back up ID, though UKPRN should always be present and preferred
 - I'd put only UKPRN in the academy summaries
your project needs a unique identifier
 - I'd call it URN as that makes more sense, IMO, and differentiates it from the other identifiers. There's an argument that being consistent makes sense, I'd lean towards UKPRN being more of a DfE specific convention and `unique reference number` is sensible for a service specific entity

Reply:
note: the above comment was to clarify the comment on the outdated code, the latest is very nearly there, though I think the TrustSummary object needs the `group_id` _as well_